### PR TITLE
Fix #941: omit tags from README to minimize spam

### LINKED ISSUE #941: Reduce commit spam by moving container information elsewhere
While I love the idea of having these containers and badges it does come at an expense of having a lot of spam in the commit log. Ideally it would be great to move these to outside the README maybe in the Github wiki?

### PR DESCRIPTION
**Pull request by rtrox**
Per title -- this PR fixes #941 by removing the image tags from the README. The result of this PR will be that the README will only see an update when a new container image is added. Otherwise, users can follow the link to the package to see available tags.

Once https://github.com/badges/shields/issues/5594 is resolved, we can look at adding tags back in via shields.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,35 +65,35 @@ For applications that need to have persistent configuration data the config volu
...
-[theme-park](https://github.com/onedr0p/containers/pkgs/container/theme-park) | stable | ghcr.io/onedr0p/theme-park |![1](https://img.shields.io/badge/1-blue?style=flat-square) ![1.16](https://img.shields.io/badge/1.16-blue?style=flat-square) ![1.16.0](https://img.shields.io/badge/1.16.0-blue?style=flat-square) ![rolling](https://img.shields.io/badge/rolling-blue?style=flat-square)
-[volsync](https://github.com/onedr0p/containers/pkgs/container/volsync) | stable | ghcr.io/onedr0p/volsync |![0](https://img.shields.io/badge/0-blue?style=flat-square) ![0.8](https://img.shields.io/badge/0.8-blue?style=flat-square) ![0.8.1](https://img.shields.io/badge/0.8.1-blue?style=flat-square) ![rolling](https://img.shields.io/badge/rolling-blue?style=flat-square)
+Container | Channel | Image
+--- | --- | ---
+[actions-runner](https://github.com/None/pkgs/container/actions-runner) | stable | ghcr.io/None/actions-runner
```
Comment (onedr0p) [inline README.md:70]: Looks like there is some rendering issue here.

Comment (rtrox) [reply README.md:70]: whoops! Good catch. It's not actually a bug, I just ran the script without setting the env var:
https://github.com/onedr0p/containers/pull/980/files/2e0b21b9ac2ff7761927e53c9a9fb02127813e17#diff-169f12ecccd420a8918cd1efe37341a1415bab93ea76d9f3482cc8465c103acdL9

I'm stepping into a meeting, but in a little bit I can rerun the script to fix the initial README content.

Comment (rtrox) [reply README.md]: Fixed!

**Comment by onedr0p** [comment]
Thanks for the updates!

**Pull request merged**